src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -1452,6 +1455,8 @@ def destroy_containers(self, containers):
                         self.test_log.info("  {}".format(error))
                         error_list.append(
                             "Error destroying container: {}".format(error))
+        elif self.skip_destroy_containers:
+            self.test_log.info("SKIP Destroying containers as requested by test")
```

Comment:
If we want to retain this messaging, we should add the `skip_destroy` attribute to the `TestContainer` class and include it i the `TestContainer.destroy()` method.